app/views/home/index.html.haml

```diff
@@ -35,7 +35,7 @@
         .row.align-items-center
           .col-lg-6.order-lg-2
             .p-5
-              =image_tag 'mf3.jpeg', class: 'circ_image'
+              =image_tag 'mf3.jpeg', class: 'circ_image', size: "800x500"
```

Comment:
Maybe the size should be defined in the css.

Reply:
want me to change circ_image to contain the boundaries in the css?

Reply:
It's just that different images might require different sizes.